src/chrome/content/rules/Jimdo.xml

```diff
@@ -1,46 +1,23 @@
 
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://www51.jimdo.com/ => https://www51.jimdo.com/: (51, "SSL: no alternative certificate subject name matches target host name 'www51.jimdo.com'")
-
 	Other Jimdo rulesets:
-
-		- Jim_static.com.xml
-
-
-	Nonfunctional subdomains:
-
-		- ^ ¹
-		- blog ²
-		- u ³
-		- support ¹
-		- web63 ⁴
-		- web40[23] ¹
-		- www ³
-
-	¹ Dropped
-	² wpengine
-	³ 403; mismatched, CN: ssl2.cdngc.net
-	⁴ Refused
-
+		-Jimdo-Server.com.xml
+		-Jimcdn.com.xml
+		-Jimstatic.com.xml
+		-Jimdo.design.xml
 -->
-<ruleset name="Jimdo.com (partial)" default_off='failed ruleset test'>
+<ruleset name="Jimdo.com">
 
 	<!--	Direct rewrites:
 				-->
-	<target host="a.jimdo.com" />
-	<target host="webmail.jimdo.com" />
-	<target host="www51.jimdo.com" />
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.jimdo\.com$" name="^JDI$" /-->
-	<!--securecookie host="^a\.jimdo\.com$" name="^PHPSESSID$" /-->
-	<!--securecookie host="^www51\.jimdo\.com$" name="^ClickAndChange$" /-->
-
-	<securecookie host="^(?:a|webmail|www51)\.jimdo\.com$" name=".+" />
-
+	<target host="jimdo.com" />
+	<target host="*.jimdo.com" />
```

Comment:
We are discouraging the use of wildcards. You should explicitly list working hosts as targets instead.

Reply:
I know but all customer sites have a subdomain from Jimdo. All subdomains use https and the certificate from Jimdo.

Reply:
Okay, can you add a comment to explain this in this case?